addons/mail/models/template_reset_mixin.py

```diff
@@ -70,15 +70,15 @@ def _override_translation_term(self, module_name, xml_ids):
             # Step 1: reset translation terms with base language file
             if '_' in lang_code:
                 base_lang_code = lang_code.split('_')[0]
-                base_trans_file = get_module_resource(module_name, 'i18n', base_lang_code + '.po')
-                if base_trans_file:
-                    translation_importer.load_file(base_trans_file, code, xmlids=xml_ids)
-
-                # Step 1.5: in case of latin america Spanish variation, load es_MX too
-                if base_lang_code == "es" and lang_code != "es_MX":
-                    mx_trans_file = get_module_resource(module_name, 'i18n', 'es_MX.po')
-                    if mx_trans_file:
-                        translation_importer.load_file(mx_trans_file, code, xmlids=xml_ids)
+                if base_lang_code == "es" and lang_code != "es_419":
```

Comment:
Shouldn' we have a tool for that ? Seems weird to have this kind of logic tripliced (does that word exist ? probably not) in code ^^

Reply:
why is there a `_override_translation_term` in mail in the first place? :thinking:

Reply:
Seems that
```
            # In case of sub languages (e.g fr_BE), load the base language first, (e.g fr.po) and
            # then load the main translation file (e.g fr_BE.po)"
```

Probably security / translation / internal "team" (we all know who it is, isn't it Mr Transifex) that did not do its job at that time.

Reply:
> Shouldn' we have a tool for that ? Seems weird to have this kind of logic tripliced (does that word exist ? probably not) in code ^^

Agree, since the logic becomes more and more complicated, I think it is the time to refactor it a litte bit to allow developers to get all relevant po files' locations with a function/method like the `_get_po_paths`.

I can create a PR for it next week.